doc/Getting-Started-Angular.html

```diff
@@ -141,12 +147,12 @@ <h4>Application UI</h4>
 <p>After login to the application, you will 
 see the sample dashboard screen:</p>
 <p>
-<img class="img-thumbnail" alt="Dashboard" height="965" src="images/dashboardV2.png" width="1235" /></p>
+<img class="img-thumbnail" alt="Dashboard" height="965" src="images/dashboardV3.png" width="1235" /></p>
 <h3 id="DocMore">More</h3>
 
 <p>Your solution is up and working.
 See
-<a href="Development-Guide-Angular.html"><span class="text-primary">development guide</span></a> document for more 
+<a href="Getting-Started-Xamarin.html"><span class="text-primary">getting started</span></a> for Xamarin application, <a href="Development-Guide-Angular.html"><span class="text-primary">development guide</span></a> document for more 
```

Comment:
There's no `Getting-Started-Xamarin` — did you mean `Development-Guide-Xamarin`? (also: [here](https://github.com/aspnetzero/documents/pull/29/files/5518739bffa047a3eda136683430351008833e1a#diff-4da194f25b0fea35173718c38fbbd369R123))

Reply:
Thank you @acjh for finding that. Fixing it.

Reply:
yes, there was a misunderstanding, I have fixed it, thanks 😄

Reply:
👍